application/views/presenter/footer.php

```diff
@@ -106,5 +131,103 @@ function checkbox() {
+        }
+        // No activity let everyone know
+        function inActive(){
+            socket.emit('userActiveChangeInApp', {"app":socket_app_name, "room":socket_active_user_list, "name":user_name, "userId":user_id, "status":false});
+        }
+
+        $(window).on("blur focus", function(e) {
+            var prevType = $(this).data("prevType");
+
+            if (prevType != e.type) {   //  reduce double fire issues
+                switch (e.type) {
+                    case "blur":
+                        inActive();
+                        break;
+                    case "focus":
+                        resetActive();
+                        break;
+                }
+            }
+
+            $(this).data("prevType", e.type);
+        });
+    });
+</script>
+
+<script type="text/javascript">
+    $(document).ready(function () {
+        var app_name_main = "<?=getAppName("") ?>";
+        push_notification_admin();
+        //setInterval(push_notification_admin, 2000);
+        socket.on('push_notification_change', (socket_app_name) => {
+            if (socket_app_name == app_name_main)
+                push_notification_admin();
+        });
+        function push_notification_admin()
+        {
+            var push_notification_id = $("#push_notification_id").val();
+
+            $.ajax({
+                url: "<?= base_url() ?>presenter/push_notification/get_push_notification_admin",
+                type: "post",
+                dataType: "json",
+                success: function (data) {
+                    if (data.status == "success") {
+                        if (push_notification_id == "0") {
+                            $("#push_notification_id").val(data.result.push_notification_id);
+                        }
+                        if (push_notification_id != data.result.push_notification_id && data.result.session_id == null) {
+                         
+                            $("#push_notification_id").val(data.result.push_notification_id);
+                            $('#push_notification').modal('show');
+                            $("#push_notification_message").text(data.result.message);
+                            
+                        }
+
+                        if (push_notification_id != data.result.push_notification_id && data.result.session_id != null)
+                        {
+                         
+                            if (typeof session_id !== 'undefined' && session_id == data.result.session_id)
+                            {
+                                $("#push_notification_id").val(data.result.push_notification_id);
+                                $('#push_notification').modal('show');
+                                $("#push_notification_message").text(data.result.message);
+                            
+                        }
```

Comment:
How will this show ONLY for presenters notifications?   This will show notifications that's sent to even attendees.

Reply:
Also in this case, you have to add some condition in the attendee side, if the notification is ONLY for presenter, DON"T show it to the attendee.